andela/ah-django-unchained-frontend
# #162948806 get article draft

#### What does this PR do?
Allows user to get his/her own article drafts.
#### Description of Task to be completed?
* [x]    Reuse UI components
* [x]    Configure Redux and thunk middleware
* [x]    Consume the `get article draft` API endpoint
* [x]    Write Tests for both Reducers and Actions.

#### How should this be manually tested?
1. Clone this repository on your local machine using
   `git clone https://github.com/andela/ah-django-unchained-frontend.git`
2. Change your working directory to the project's directory using
   `cd ah-django-unchained-frontend`
3. Change to the specific branch `ft-create-draft-article-162948806 ` using
   `git checkout ft-view-draft-162948806 `
4. Create a `.env` file and inside it add this line `REACT_APP_BASE_URL=app-base-url` where `app-base-url ` is the root url of the backend API
5. Run `npm install` to install all the modules listed in the `package.json` file.
6. Run `npm start` command to start the application.
7. The application should start successfully. On the browser, navigate to the URL 
login navigate to  `http://localhost:3000/login`
You will be redirected to view drafts `http://localhost:3000/draft`

Screenshot

![image](https://user-images.githubusercontent.com/18655501/53473301-1c32f200-3a7b-11e9-83e8-14c86de28ffa.png)

#### Link to PT story[#162948806](https://www.pivotaltracker.com/story/show/162948806)
